tools/fetch-external.sh

```diff
@@ -80,9 +80,9 @@ cd pages/learn/more/masterclasses/ && {
 cd shared/projects/ && {
   echo "Name|Description
 ---|---" | tee balena-labs-projects.md balena-example-projects.md balenablocks.md >/dev/null
-  curl https://api.github.com/orgs/balenalabs/repos?per_page=30 | ../../node_modules/node-jq/bin/jq -r 'sort_by(-.stargazers_count) |  (.[] | [.name,.html_url,.description] | "[\(.[0])](\(.[1]))|\(.[2] // "")") ' >>balena-labs-projects.md
-  curl https://api.github.com/orgs/balena-io-examples/repos?per_page=100 | ../../node_modules/node-jq/bin/jq -r 'sort_by(-.stargazers_count) |  (.[] | [.name,.html_url,.description] | "[\(.[0])](\(.[1]))|\(.[2] // "")") ' >>balena-example-projects.md
-  curl https://api.github.com/orgs/balenablocks/repos?per_page=30 | ../../node_modules/node-jq/bin/jq -r 'sort_by(-.stargazers_count) |  (.[] | [.name,.html_url,.description] | "[\(.[0])](\(.[1]))|\(.[2] // "")") ' >>balenablocks.md
+  curl https://api.github.com/orgs/balenalabs/repos?per_page=30 | jq -r 'sort_by(-.stargazers_count) |  (.[] | [.name,.html_url,.description] | "[\(.[0])](\(.[1]))|\(.[2] // "")") ' >>balena-labs-projects.md
```

Comment:
@cmfcruz Wouldn't the contributors might face issues with this later down the road as jq is available on all popular OS's but one has to install it first. This was the reason we didn't want to go for the OS package `jq` rather went `node-jq`. Let me know if it wouldn't matter.